src/search.cpp

```diff
@@ -1019,7 +1019,7 @@ Value Search::Worker::search(
                 lmrDepth = std::max(lmrDepth, 0);
 
                 // Prune moves with negative SEE (~4 Elo)
-                if (!pos.see_ge(move, -24 * lmrDepth * lmrDepth))
+                if (pos.see(move) < -24 * lmrDepth * lmrDepth)
```

Comment:
> No, there is no function in master (nor in this PR) to calculate the SEE value of a move, we can only compare that value to an upper bound or a lower bound via see_ge()

@snicolet I'm referring to here, aren't we calling the function again on the move, if we are passing move to the function aren't we calculating the result for such move in this position.. is this call not redundant to the ones in the movepicker?

Reply:
if we save the result S. and reuse it?

Reply:
this is not a function, this is just syntaxic sugar...

```
pos.see(move) < A     
```
is strictly equivalent to
```
!pos.see_ge(move, A)
```

Reply:
Ok, I see so it's like alpha-beta?
So you are saying that the value is not exact for some reason,
is it not possible to match bounds at least, just something like the saved bounds in TT, where we at least match windows/thresholds?
`&& (ttData.bound & (ttData.value >= beta ? BOUND_LOWER : BOUND_UPPER)))`

Reply:
i.e. if a value fails high over +3000, is calculating see_ge of threshold -90000 relevant?